mixins/visual/responsive-visual.vue

```diff
@@ -110,13 +140,20 @@ export default
 
 		# Make the config object for the create function by keeping all data and
 		# props except for replacing landscape and portrait with the asset itself
+		# and removing preload tag in case it was added
 		makeConfig: (image, video) ->
 			return unless image or video
+
+			# If responsive, preloading will be handled by this component, not
+			# the child cloak-visual
+			preload = if @isResponsiveImage then false else @$props.preload
+
 			on: loaded: => @$emit 'loaded'
 			props: {
 				...@$props
 				image
 				video
+				preload
```

Comment:
I think you should auto turn off lazyloading, too, when preloading, [like here](https://github.com/BKWLD/cloak-visual/blob/1e9ccf0ee0fa9db669b24d0ad07eba640a8d9e61/components/responsive.vue#L157-L160).

I added this stuff this morning in https://github.com/BKWLD/cloak-visual/pull/24, realizing that responsive visuals wouldn't auto set the preload attributes like we want.

Reply:
just pushed a [change](https://github.com/BKWLD/cloak/pull/63/commits/cae51c314da13a782571649615f4e1734e672fab) to turn off lazyloading when preloading.